designpage.html

```diff
@@ -95,16 +95,16 @@
 			</div>
 			<div class="containthumbs">
 			<div>
-				<img class="img2" onclick="displayImg1Change()"  src="images/image-product-1-thumbnail.jpg"> 
+				<img id="img1" class="img2" onclick='displayImgChange("img1")'  src="images/image-product-1-thumbnail.jpg"> 
```

Comment:
WITH RESPECT TO THE FIRST COMMENT;

Like see this is the main(full) display image and its src is linking the full image file not the thumbnail